src/audio/CMakeLists.txt

```diff
@@ -121,8 +101,8 @@ check_optimization(hifi3 -mhifi3 -DOPS_HIFI3)
 set(sof_audio_modules volume src)
 
 # sources for each module
-set(volume_sources volume.c volume_generic.c)
-set(src_sources src.c src_generic.c)
+set(volume_sources volume/volume.c volume/volume_generic.c)
```

Comment:
Would be better to not handle f.e. volume.c on 2 different levels for different builds. We should have some kind of function here that is called in subdirs maybe. I can refactor that part later. We can merge it like it is now cos it's step in the good direction and it's better to have smaller PRs anyway.